code/API_definitions/qod-api.yaml

```diff
@@ -433,6 +472,21 @@ components:
             - expiresAt
             - qosStatus
 
+    SessionInfoList:
+      description: Paged response of active sessions created by the user
+      type: object
+      properties:
+        perPage:
+          $ref: "#/components/schemas/PerPage"
+        page:
+          $ref: "#/components/schemas/Page"
+        sessions:
+          description: Array of sessions to be returned.
+          type: array
+          items:
+            type: object
+            $ref: "#/components/schemas/SessionInfo"
```

Comment:
The question is what should happen if no sessions are available.
Version 1
Server returns 404
In this case the parameter sessions in SessionInfoList should be mandatory, and it contains at least 1 session

Version 2
Server returns 200 and the sessions parameter is missing
No 404 should be defined.

Version 3
Server returns 200 and the sessions parameter is mandatory and contains 0 elements
No 404 should be defined.

Reply:
Aaa yeah, I hadn't thought about this situation.. You are right, returning 404 in this case is not very good since it is not really error. And I also think that "sessions" parameter should be mandatory and if there are no active sessions it just should be an empty array.

Reply:
I also commit on Version 3